tools/version_update.py

```diff
@@ -5,7 +5,7 @@
 pom = open("pom.xml")
 dom = parse(pom)
 pom.close()
-mv = dom.getElementsByTagName('motech.version')
+mv = dom.getElementsByTagName('project.version')
```

Comment:
I think that this script is now unnecessary, it was only creating a commit with bump up of _motech.version_ property value.